src/Components/ExtractLevelsMenu.jsx

```diff
@@ -33,6 +33,11 @@ export default function ExtractLevelsMenu({listOfOptions, icon, title}) {
     setAnchorEl(null)
   }
   const viewer = useStore((state) => state.viewerStore)
+  // ADDED
```

Comment:
remove the comments. 
for the review purposes leave the comments in the PR.

Reply:
Addressed. Removed logs.